.circleci/config.yml

```diff
@@ -301,6 +301,10 @@ jobs:
           name: hadoLint_openjdk_latest
           command: |
             docker run --rm -i hadolint/hadolint < docker/openjdk-latest/Dockerfile
+      - run:
+          name: hadoLint_openjdk_latest
+          command: |
+            docker run --rm -i hadolint/hadolint < docker/openjdk-latest-debug/Dockerfile
```

Comment:
@jflo we are no longer using circle-ci correct?

Reply:
okay, I removed the circleci changes . @jflo  let me know If you want me to add it back.